### DISCUSSION TITLE
Does normcap works on Hyprland or is it being worked on?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [dynobo/normcap#573] [Q&A] [answered]
Saw two github issues - https://github.com/dynobo/normcap/pull/532 https://github.com/dynobo/normcap/issues/375

As v5.0 i thought it would work but this in my system, normcap opens its own seprate window ->

![image](https://github.com/dynobo/normcap/assets/75798544/ae20aaa7-6de1-401a-91ae-6b11a092c8fd)

**Comment by dynobo** [comment, marked as answer]
Hi @KAGEYAM4, legitimate question!  

I _tried_ to add support for it (#532), but didn't yet succeed (that's why #375 is still open).

The main hurdle is, that [I haven't found an easy way to _run_ Hyprland myself for testing](https://github.com/dynobo/normcap/issues/375#issuecomment-1752104306) and I am therefore dependent on Hyprland-Users like you to test stuff.

Right now, I'm stalled as [I've been waiting for input for quite a while](https://github.com/dynobo/normcap/issues/375#issuecomment-1752180468): If _you_ want to help getting the Hyprland support working, it would be great if you download the appimage, run `./NormCap-0.5.2-x86_64.AppImage -v debug` in the terminal on your system, and report the output in #375. :slightly_smiling_face:

**Reply by KAGEYAM4** [reply]
Ok, i just posted the output.
